clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -570,11 +570,14 @@ def _load_fileset_from_server(self, identifier: NameIdentifier) -> Fileset:
         :return The fileset
         """
         catalog: Catalog = self._client.load_catalog(
```

Comment:
the "load_catalog" method has been refactored by me yesterday; I guess this couldn't work now, should change to use catalog name as input parameter.

Reply:
Ah yes, I missed that.